ui/app/AppLayouts/Browser/BrowserLayout.qml

```diff
@@ -479,6 +482,12 @@ Rectangle {
                     sslDialog.enqueue(error);
                 }
 
+                onJavaScriptDialogRequested: function(request) {
+                    request.accepted = true;
+                    var dialog = jsDialogComponent.createObject(browserWindow, {"request": request});
```

Comment:
I think that you can just do the following instead of creation a property
```qml
var dialog = JSDialogWindow.createObject(browserWindow, {"request": request});
```

Reply:
```
TypeError: Property 'createObject' of object [object Object] is not a function
```
U.U

Reply:
Ah right, my bad, it was using `Component{}` so it's basically the same as you